charts/enterprise-metrics/templates/graphite-querier-dep.yaml

```diff
@@ -0,0 +1,146 @@
+{{- if .Values.graphite.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "enterprise-metrics.fullname" . }}-graphite-querier
+  labels:
+    app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+    chart: {{ template "enterprise-metrics.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- toYaml .Values.graphite_querier.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.graphite_querier.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+      release: {{ .Release.Name }}
+  strategy:
+    {{- toYaml .Values.graphite_querier.strategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+        # The name label is important for cortex-mixin compatibility which expects certain names for services.
+        name: graphite-querier
+        target: querier
+        release: {{ .Release.Name }}
+        {{- with .Values.graphite_querier.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      annotations:
+{{- if .Values.useExternalConfig }}
+        checksum/config: {{ .Values.externalConfigVersion }}
+{{- else }}
+        checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+{{- end}}
+        {{- with .Values.graphite_querier.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ template "enterprise-metrics.serviceAccountName" . }}
+    {{- if .Values.graphite_querier.priorityClassName }}
+      priorityClassName: {{ .Values.graphite_querier.priorityClassName }}
+    {{- end }}
+      securityContext:
+        {{- toYaml .Values.graphite_querier.securityContext | nindent 8 }}
+      initContainers:
+        {{- toYaml .Values.graphite_querier.initContainers | nindent 8 }}
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end}}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - -target=graphite-querier
+            - -config.file=/etc/enterprise-metrics/enterprise-metrics.yaml # TODO(jesus.vazquez) Remove when metrics enterprise v1.6.2 is released
```

Comment:
which change in `1.6.2` is the reason that this can be removed then?

Reply:
You are right that the config file could still be used as a preferred way to initialize the proxy over command arguments so I'll leave it and remove the TODO comment.